src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < balls.length; i++) {
```

Comment:
rewrite without an array and using one for loop

Reply:
no need to store it in the array, use right away -  `System.out.println(lottery.getRandomBall());`